OnlineLibrary.BLL/Interfaces/IAuthorService.cs

```diff
@@ -2,16 +2,20 @@
 using System.Collections.Generic;
 using System.Text;
 
-namespace BookAuditNLayerApp.BLL.Interfaces
+namespace OnlineLibrary.BLL.Interfaces
 {
     public interface IAuthorService<T> where T : class
     {
-        List<T> GetAllAuthors();
+        List<T> GetAllAuthors(IDataExportService dataExportService);
 
         int CreateAuthor(T author);
         // Returns Id.
 
         bool IsAuthorIdExists(int? authorId);
 
+        bool IsAuthorsExists(List<int> authorsId);
```

Comment:
_bool IsAuthorIdExists(int? authorId)_
 You can create one method  _bool IsAuthorIdExists(params int[] authorId)_ to avoid 2 IsAuthorIdExists methods